charts/ccsm-helm/README.md

```diff
@@ -238,9 +238,47 @@ Information about Tasklist you can find [here](https://docs.camunda.io/docs/comp
+| | `service.port` |  Defines the port of the service, where the identity web application will be available | `80` |
+| | `service.annotations` |  Can be used to define annotations, which will be applied to the identity service | `{}` |
+| | `resources` |  Configuration to set request and limit configuration for the container [request and limit configuration for the container](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) | `requests:`<br>`  cpu: 600m`<br> `  memory: 400Mi`<br>`limits:`<br> ` cpu: 2000m`<br> ` memory: 2Gi` |
+| | `env` |  Can be used to set extra environment variables in each identity container | `[]` |
+| | `command` |  Can be used to override the default command provided by the container image. See [override the default command provided by the container image](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/) | |
+| | `extraVolumes` |  Can be used to define extra volumes for the identity pods, useful for tls and self-signed certificates | `[]` |
+| | `extraVolumeMounts` |  Can be used to mount extra volumes for the identity pods, useful for tls and self-signed certificates | `[]` |
+| | `keycloak` |  Configuration, for the keycloak dependency chart which is used by identity | |
+| | `keycloak.auth` |  Authentication parameters - see [admin-credentials](https://github.com/bitnami/bitnami-docker-keycloak#admin-credentials) | |
+| | `keycloak.auth.adminUser` |  Defines the keycloak administrator user | 'admin' |
+| | `keycloak.auth.existingSecret` |  Can be used to reuse an existing secret containing authentication information. See [manage-passwords](https://docs.bitnami.com/kubernetes/apps/keycloak/configuration/manage-passwords/) for more details. | `""` |
+| | `serviceAccount` |  Configuration for the service account where the identity pods are assigned to | |
+| | `serviceAccount.enabled` |  If true, enables the identity service account | `true` |
+| | `serviceAccount.name` |  Can be used to set the name of the identity service account | `` |
+| | `serviceAccount.annotations` |  Can be used to set the annotations of the identity service account | `{}` |
+| | `ingress.enabled` | If true, an ingress resource is deployed with the Identity deployment. Only useful if an ingress controller is available, like nginx. | `false` |
+| | `ingress.className` | Defines the class or configuration of ingress which should be used by the controller | `nginx` |
+| | `ingress.annotations` | Defines the ingress related annotations, consumed mostly by the ingress controller | `ingress.kubernetes.io/rewrite-target: "/"` <br/> `nginx.ingress.kubernetes.io/ssl-redirect: "false"` |
+| | `ingress.path` | Defines the path which is associated with the Identity service and port https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules | `/` |
```

Comment:
```suggestion
| | `ingress.path` | Defines the path and port which is associated with the Identity service https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules | `/` |
```

Reply:
Not sure whether this is true :thinking:

Reply:
lol sorry for the confusion I thought you added it :D got confused by the suggestions view